test/excludes/CommentTest.rb

```diff
@@ -0,0 +1,2 @@
+exclude :test_remove_comment_from_column, "The test fails because ALTER COLUMN TYPE is not supported."
```

Comment:
ALTER COLUMN TYPE is supported with the setting `SET enable_experimental_alter_column_type_general = true`. we could turn that setting on for these tests.

Reply:
I tried to run the tests with this set and they ended up failing for another reason. In both tests, they change a column type from `bool` to `character varying`, and once a column is altered, the comment on it can't be changed.

I ended up modifying the tests to not change the column type since these are mainly testing that comments can be removed/changed through migrations. 

I can open an issue about the inability to modify comments on altered columns if it's not already tracked.

Reply:
oh nice find! please do open that issue, that sounds like an unknown bug. if you can post SQL to reproduce it that would help.

Reply:
https://github.com/cockroachdb/cockroach/issues/65568